src/index.ts

```diff
@@ -145,13 +150,39 @@ export default class ServerlessConventions {
     // Check that the service name does not contain the word "service"
     if (serviceName.toLowerCase().includes('service')) {
       errors.push(
-        `Warning: Service name should not include the word "service"`
+        `Warning: Service name "${serviceName}" should not include the word "service"`
       );
     }
 
     // Check the length of the service name is not greater than x
     if (serviceName.length > 23) {
-      errors.push(`Warning: Service name must be less than 23 characters`);
+      errors.push(
+        `Warning: Service name "${serviceName}" must be less than 23 characters`
+      );
+    }
+
+    return errors;
+  }
+
+  // Stage name validation
+  // Must contains only lower case alphabet characters
+  // Must be only 3 characters
+  checkStageName(service: Service): Array<string> {
+    let errors: Array<string> = [];
+    const stageName = service.provider.stage;
```

Comment:
Is it possible for `stageName` to be something other than a string here? If so we should default to an empty string

Reply:
It seems impossible to me for `service.provider.stage` to be something else here. I tried with `stage.provider.stage` as an empty string & Serverless automatically throw me an error . Their `stage` restriction is `^[a-zA-Z0-9-]+$`

![image](https://user-images.githubusercontent.com/108910975/190530726-00c7c8a9-dbe7-4245-b26b-81bc14a9734e.png)

In addition, `provider.stage` type is `string` in Serverless.Service class:

![image](https://user-images.githubusercontent.com/108910975/190530596-0cb254c8-f8c1-4012-8688-24d9baba367b.png)